system-tests/dashboard/test-dashboard.js

```diff
@@ -0,0 +1,250 @@
+
+      label = getAllocationElementFor("CPU Allocation");
+      label.should(function($label) {
+        const newValue = parseFloat($label.text());
+        expect(newValue).to.eq(initialValue + 0.1);
+      });
+    });
+
+    it("increments memory usage when service is started", function() {
+      let label = getAllocationElementFor("Memory Allocation");
+      let initialValue;
+      label.should(function($label) {
+        initialValue = parseFloat($label.text());
+      });
+
+      createService();
+      cy.visitUrl("dashboard");
+
+      label = getAllocationElementFor("Memory Allocation");
+      label.should(function($label) {
+        const newValue = parseFloat($label.text());
+        expect(newValue).to.eq(initialValue + 10);
+      });
+    });
+
+    it("increments disk usage when service is started", function() {
+      let label = getAllocationElementFor("Disk Allocation");
+      let initialValue;
+      label.should(function($label) {
+        initialValue = parseFloat($label.text());
+      });
+
+      createService();
+      cy.visitUrl("dashboard");
+
+      label = getAllocationElementFor("Disk Allocation");
+      label.should(function($label) {
+        const newValue = parseFloat($label.text());
+        expect(newValue).to.eq(initialValue + 10);
+      });
+    });
+
+    it("new service shows in services list on dashboard when started", function() {
+      createService();
+      cy.visitUrl("dashboard");
+
+      cy
+        .get(".panel-header")
+        .contains("Services Health")
+        .parents(".panel")
+        .find("li")
+        .contains(SERVICE_NAME);
+    });
+
+    it("new service increments task count in dashboard", function() {
+      let taskCountElement = getTaskCountElement();
+      let tasksCountInitial;
+      taskCountElement.should(function($label) {
+        tasksCountInitial = parseFloat($label.text());
+      });
+
+      createService();
+      cy.visitUrl("dashboard");
+
+      taskCountElement = getTaskCountElement();
+      taskCountElement.should(function($tasksCount) {
+        const newValue = parseFloat($tasksCount.text());
+        expect(newValue).to.eq(tasksCountInitial + 1);
+      });
+    });
+  });
+
+  describe("Delete service tests", function() {
+    beforeEach(() => {
+      cy.visitUrl("dashboard");
+      let label = getAllocationElementFor("CPU Allocation");
+      label.should(function($label) {
+        expect(parseFloat($label.text())).to.equal(0);
+      });
+
+      createService();
+      cy.visitUrl("dashboard");
+      label = getAllocationElementFor("CPU Allocation");
+      label.should(function($label) {
+        expect(parseFloat($label.text())).to.equal(0.1);
+      });
+    });
+
+    afterEach(() => {
+      cy.window().then(win => {
+        win.location.href = "about:blank";
+      });
+    });
+
+    it("decrements CPU usage when service is stopped", function() {
+      let label = getAllocationElementFor("CPU Allocation");
```

Comment:
Also can remove these because `beforeEach()` I'm asserting there is a single service running.